src/channelape/controller/ChannelApeActionsController.ts

```diff
@@ -10,36 +10,36 @@ const PARSE_INT_RADIX = 10;
 
 export default abstract class ChannelApeActionsController {
   protected logger: Logger;
-  private updateHealthCheckInterval?: NodeJS.Timer;
 
   constructor(loggerName: string, protected channelApeClient: ChannelApeClient) {
     this.logger = new Logger(loggerName, Secrets.env.LOG_LEVEL);
   }
 
-  protected abstract processAction(businessId: string, actionId: string): void;
+  protected abstract processAction(businessId: string, actionId: string): Promise<any>;
```

Comment:
Can we make this `Promise<void>`? What reason is there to return a value from this function?